keps/sig-storage/20190530-pv-health-monitor.md

```diff
@@ -537,6 +538,13 @@ This optional service is out of scope for this KEP but may be introduced in a fu
 This option is not in the main proposal because it is a push-based method while CSI uses pull-based method.
 
 ## Graduation Criteria
+### Alpha
+* Initial feature implementation, including:
+  * External controller volume health monitoring.
+  * External node agent volume health monitoring.
+* Implementation in the csi-mock driver.
+* Add corresponding unit tests.
```

Comment:
The tests listed there are specifying error codes, which I believe we removed from the CSI spec proposal. Do we need to update the KEP with the csi spec updates?

Reply:
Will do.

Reply:
Done